### PR TITLE
DCS-667 PSH redirect to bvl service

### PR DESCRIPTION
**Pull request by GurnankCheema**

### CODE REVIEW

File: backend/setupRedirects.js [hunk 1]
```diff
@@ -22,5 +22,25 @@ module.exports = () => {
     res.redirect(301, `/prisoner/${req.params.offenderNo}/incentive-level-details`)
   )
 
+  if (config.app.featureFlags.redirectToBookingVideoLinkEnabled) {
+    const bvlRoutes = [
```
Comment (andrewrlee) [inline backend/setupRedirects.js:26]: So the idea would be to move this into the new `setupBvlRoutes.js`
```js
if (enabled) {
  // just add redirect for `/videolink` on its own
} else {
 // register all routes
}
```
When we remove vlb entirely from here we might want to semi-permanently add the `/videolink` redirect here.

Comment (GurnankCheema) [reply backend/setupRedirects.js:26]: i see.

File: backend/tests/setupBvlRoutes.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    expect(path).toBe('/videolink')
+
+    const res = {
+      redirect: jest.fn(),
+    }
+
+    handler(null, res)
+
+    expect(res.redirect).toHaveBeenCalledWith(config.apis.bookVideoLink.url)
+  })
```
Comment (andrewrlee) [inline backend/tests/setupBvlRoutes.test.js:33]: This is a bit gnarly but think it's probably acceptable given we'll be removing and simplifying the video link booking code and just leaving a simple redirect that we can but in with the standard redirects

**Review by JamieDraperUK — DISMISSED** [review]
LGTM

**Review by JamieDraperUK — APPROVED** [review]

**Pull request merged**